modules/core/src/weaver/Expectations.scala

```diff
@@ -83,6 +83,12 @@ object Expectations {
 
   trait Helpers {
 
+    /**
+     * Expect macros
+     */
+    def expect = new Expect
+    def assert = new Expect
```

Comment:
Nice - I've fallen into a pattern of re-defining those in custom traits that are not part of the suite